libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/BabbageFeatures.hs

```diff
@@ -258,7 +258,8 @@ defaultPPs =
     MaxTxExUnits $ ExUnits 1000000 1000000,
     MaxBlockExUnits $ ExUnits 1000000 1000000,
     ProtocolVersion $ ProtVer 7 0,
-    CollateralPercentage 100
+    CollateralPercentage 1,
+    AdaPerUTxOWord (Coin 5)
```

Comment:
Is it a percentage,or is it a rational (or float) between 0 and 1?

Reply:
it is a number of lovelace